lib/consign.js

```diff
@@ -66,6 +66,11 @@ function Consign(options) {
  */
 
 Consign.prototype._setLocations = function(parent, entity, push) {
+  if (!entity) {
+    this._log(['! Entity name not provided or not exists', location], 'error');
```

Comment:
Please Chand the following:

'... provided or not exists' to '... provided or doesn't exist' cheers :)